src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas3/ReportsController.kt

```diff
@@ -125,11 +125,17 @@ class ReportsController(
     validateRequestedDates(startDate, endDate)
   }
 
+  @SuppressWarnings("ThrowsCount")
   private fun validateRequestedDates(startDate: LocalDate, endDate: LocalDate) {
     when {
-      startDate.isAfter(endDate) -> throw BadRequestProblem(errorDetail = "Start Date $startDate cannot be after End Date $endDate")
+      startDate.isAfter(endDate) || startDate.isEqual(endDate) ->
+        throw BadRequestProblem(invalidParams = mapOf("$.startDate" to "afterEndDate"), errorDetail = "Start Date $startDate cannot be after End Date $endDate")
```

Comment:
The message should be different if the start date is the same as end date

Reply:
No, as this is the same error: start date is on or after end date. The message is not needed at all, in fact -- for consistency with the rest of the API, the error should only be thrown with:

```suggestion
        throw BadRequestProblem(invalidParams = mapOf("$.startDate" to "afterEndDate"))
```

Reply:
I've updated the PR to remove the error details